platform/solr/solr-factory/src/test/groovy/org/codice/solr/factory/impl/HttpSolrClientFactorySpec.groovy

```diff
@@ -178,8 +186,6 @@ class HttpSolrClientFactorySpec extends Specification {
       factory.newClient(null)
 
     then:
-      def e = thrown(IllegalArgumentException)
-
-      e.message.contains("invalid null Solr core")
+      thrown(IllegalArgumentException)
```

Comment:
The additional check was making sure that we were failing for that reason. `IllegalArgumentException` is so generic that you never know really who throws it

Reply:
The `Args` always throws that exception. We can where it was thrown by the message on the exception:

` Args.notEmpty(coreName, "Cannot create Solr client. Missing core name.");`

Reply:
By definition, `IllegalArgumentException` or any other runtime exceptions could be thrown anywhere for any reasons. This could be thrown by code you invoked so the fact is that you have little confidence that it came because of that exact check.